docs/src/content/docs/book/maps.mdx

```diff
@@ -360,6 +360,21 @@ contract Example {
 }
 ```
 
+### Convert from a `Cell`, `.fromCell()` {#fromcell}
+
+<Badge text="Available since Tact 1.6 (not released yet)" variant="tip" size="medium"/><p/>
+
+To convert a [`Cell{:tact}`][cell] type back to a map, use the `.fromCell(){:tact}` [method](/book/functions#extension-function).
+
+```tact
+// Suppose we have a Cell
+let cell: Cell = ...;
+
+// And we want to initialize a map variable from it
+let fizz: map<Int, Int> = emptyMap();
+fizz.fromCell(cell);
```

Comment:
hmmm, this should be consistent with what we have for structs:

```tact
struct Fizz { foo: Int }
let fizzCell = ...
let parsedFizz: Fizz = Fizz.fromCell(fizzCell);
```

So I would expect something like:

```tact
let fizz = map<Int, Int>.fromCell(cell);
```

This makes parsing harder but it seems it's too late to do it differently